any/when_agegroup.md

```diff
@@ -0,0 +1,17 @@
+# when_agegroup
+
+This expression assigns a value of a certain age group based on the value of the column "age".
+
+## Use cases
+
+To categorize your age data into age groups for easier analysis and visualization.
+
+## Workflow
+
+Create a new field of type "String" in your feature layer and name it "AgeGroup". In the "field calculator" create the following Arcade expression:
+
+when($feature["applicant_age"]<15,'Under 15',$feature["applicant_age"]>=15 && $feature["applicant_age"]<25,'16-25','NA')
```

Comment:
I would format this for readability. Be sure to wrap it in a code block using three `` on each side of the snippet to pick up formatting. I would also use a `var` to extract the filed name so it's easier for users to copy and paste their own field with minimal effort. Like this...

```suggestion
// replace "applicant_age" with the name of a number
// field in your layer
var numberField = $feature["applicant_age"];

When(
  numberField < 15, 'Under 15',
  numberField >= 15 && numberField < 25, '16-25', 
'NA' );
```

```js
// replace "applicant_age" with the name of a number
// field in your layer
var numberField = $feature["applicant_age"];

When(
  numberField < 15, 'Under 15',
  numberField >= 15 && numberField < 25, '16-25', 
'NA' );
```